pkg/domain/entities/params/generator/generator.go

```diff
@@ -0,0 +1,174 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"go/ast"
+	"go/parser"
+	"go/token"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strings"
+	"text/template"
+)
+
+var bodyTmpl = `package {{.PackageName}}
+
+import (
+{{range $import := .Imports}}	{{$import}}
+{{end}}
+)
+`
```

Comment:
I would add a comment that this file is auto-generated and a timestamp for when it was written.